libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -414,14 +419,40 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         return referencePointConfigured;
     }
 
+    private getLocationProperties(locationItem: IBucketItem) {
+        // for tooltip, prefer standard text display form (whose type is `undefined`) over geo or hyperlink display forms
+        const { dfRef } = locationItem;
+        const tooltipDfRef =
+            locationItem.displayForms?.find((displayForm) => !displayForm.type)?.ref || dfRef;
+        const tooltipText = isUriRef(tooltipDfRef) ? tooltipDfRef.uri : tooltipDfRef.identifier;
+
+        if (this.backendCapabilities.supportsSeparateLatitudeLongitudeLabels) {
+            const latitudeDfRef = locationItem.displayForms?.find(
+                (displayForm) => displayForm.type === "GDC.geo.pin_latitude",
+            )?.ref;
+            const longitudeDfRef = locationItem.displayForms?.find(
+                (displayForm) => displayForm.type === "GDC.geo.pin_longitude",
+            )?.ref;
+            const latitude = isUriRef(latitudeDfRef) ? latitudeDfRef?.uri : latitudeDfRef?.identifier;
+            const longitude = isUriRef(longitudeDfRef) ? longitudeDfRef?.uri : longitudeDfRef?.identifier;
+            return {
+                tooltipText,
+                latitude,
+                longitude,
```

Comment:
This also causes these two props to be saved in persisted insight. Not sure if this is OK or not.

<img width="608" alt="image" src="https://user-images.githubusercontent.com/1880148/200347468-59d745d6-b6e6-4dcd-926c-a06325f25a9b.png">